ghost_qualifier.c

```diff
@@ -0,0 +1,10 @@
+int a ;                      // a non-ghost location
```

Comment:
Normally, example C files should be put into one of the `GOOD` and `BAD` list in the `Makefile`, depending on whether they are expected to fail.